src/WebJobs.Script/OutOfProc/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -211,6 +211,27 @@ public async Task InitializeAsync(IEnumerable<FunctionMetadata> functions)
             }
         }
 
+        public async Task ShutdownAsync()
+        {
+            _logger.LogDebug("Waiting for FunctionDispatcher to shutdown");
+            Task timeoutTask = Task.Delay(5000);
+            IEnumerable<ILanguageWorkerChannel> workerChannels = await GetInitializedWorkerChannelsAsync();
+            foreach (ILanguageWorkerChannel workerChannel in workerChannels)
+            {
+                _logger.LogDebug($"Waiting for invocations of '{workerChannel.Id}' to drain");
+                Task drainTask = workerChannel.DrainInvocationsAsync();
+                Task completedTask = await Task.WhenAny(drainTask, timeoutTask);
+                if (completedTask.Equals(timeoutTask))
+                {
+                    _logger.LogDebug($"Stopping FunctionDispatcher timed out");
+                }
+                else
+                {
+                    _logger.LogDebug("Draining of FunctionDispatcher completed");
+                }
```

Comment:
nit: More descriptive log message. 
`Draining invocations from language worker channels completed. Shutting down nameof(RpcFunctionInvocationDispatcher)`

Reply:
I changed the log messages